### ISSUE TITLE
Consider adding attributions

### ISSUE THREAD

**Issue by vinber** [thomersch/OpenLoveMap#18]
@ openstreetmap contributors and a link to osm.org/copyight ?

![image](https://user-images.githubusercontent.com/890869/177525552-474adaca-d35b-4e9e-aeea-3c2f5f851be1.png)

**Comment by thomersch** [comment]
Yeah, the original copyright info page is gone, but the referenced tile server doesn't render anyway. We could switch to some other tile layer and adjust the copyright. PRs welcome
